debug-tools/Cargo.toml

```diff
@@ -7,6 +7,5 @@ publish = false
 
 [dependencies]
 framework = { path = "../framework" }
-embedded-graphics = { git = "https://github.com/embedded-graphics/embedded-graphics.git"}
-embedded-graphics-simulator = { git = "https://github.com/embedded-graphics/simulator.git"}
-sdl2 = "0.32.2"
\ No newline at end of file
+embedded-graphics = "0.7.0-beta.1"
+embedded-graphics-simulator = "0.3.0-alpha.2"
```

Comment:
I've changed these dependencies to regular dependencies to make it easier to use a local e-g version for testing. This way you can simply change the `patch` section in the workspace `Cargo.toml` to use `path` dependencies instead of the git dependencies. Trying to add a `patch` section for the git dependencies didn't work for some reason.